Frontend/src/ContextMenu.svelte

```diff
@@ -1,154 +1,67 @@
-<script>
-    //NOTE: Add in Font Awesome later
-    let position = { x: 0, y: 0 };
 
-    let menu = { w: 0, h: 0 };
 
-    let browser = { w: 0, h: 0 };
+<script>
+    export let x;
+    export let y;
+    export let isVisible = false;
+    export let item;
 
-    let showMenu = false;
+    let editItem = () => alert(`Edit: ${Object.values(item)[0]}`);
     
-    let content;
-
-    function rightClickContextMenu(e){
-        showMenu = true;
-        browser = {
-            w: window.innerWidth,
-            h: window.innerHeight
-        };
-
-        position = {
-            x: e.clientX,
-            y: e.clientY
-        };
-    }
-
-
-    function offMenuClick(e){
-        showMenu  = false;
-    }
-
-    function getContextMenuDimension(node){
-        let height = node.offsetHeight;
-        let width = node.offWidth;
-        menu = {
-            h: height,
-            w: width
-        }
-    }
-
-    function addItem(){
-        content.textContent = "Add and item..."
-    }
-    function editItem(){
-        content.textContent = "Printed..."
-    }
-    function removeItem(){
-        content.textContent = "Removed..."
-    }
-    function setting(){
-        content.textContent = "Settings..."
-    }
+    let dupeItem = () => alert(`Duplicate: ${Object.values(item)[0]}`);
+    
+    let removeItem = () => alert(`Removed: ${Object.values(item)[0]}`);
+    // Note: These functions will need to change backend data, these alerts are 
+    //just to show what the on-screen messages may look like/show some effect
 
     let menuItems = [
         {
-            'name': 'addItem',
-            'onClick': addItem,
-            'displayText': "Add Item",
+            'name': 'editItem',
+            'onClick': editItem,
+            'displayText': "Edit",
             'icon': 'font awesome'
```

Comment:
I'm assuming 'font awesome' wasn't what you wanted for your icon

Reply:
It was more so a reminder to use the tool later.

Reply:
Gotcha. I would just rip it out and put it back later, should you need icons. Definitely wouldn't recommend having reminders in the data

Reply:
Good point. I did not realize how confusing it appeared. I'll delete them.